external-crates/move/crates/move-vm-config/src/verifier.rs

```diff
@@ -66,6 +69,7 @@ impl Default for VerifierConfig {
             max_idenfitier_len: Some(DEFAULT_MAX_IDENTIFIER_LENGTH),
             allow_receiving_object_id: true,
             reject_mutable_random_on_entry_functions: true,
+            bytecode_version: VERSION_MAX,
```

Comment:
How does this work when we cut a version?

Reply:
Normally when cutting an execution version, however when adding a new `bytecode_version` we will need to manually fix the `bytecode_version` in previous versions to the particular `VERSION_MAX` at the time they were cut. Overall I don't view this as too big of an issue though since when adding a new bytecode version a bunch of stuff needs to change all over the place anyways.